docs/filter_descriptions.md

```diff
@@ -172,65 +174,70 @@ BRNG = Broadcast Range
 
 ![Broadcast Range Anomaly 1](media/RANG_293_0046_syncloss2.jpg) ![Broadcast Range Anomaly 2](media/brng_seattleparade.jpg)
 
-## 8\. Crop Width (CropW) {#cropw}
+## Crop Width (CropW) {#cropw}
 
 This filter detects portions on either side of your video which may contain not contain any picture data. In the case of pillarboxing, for example, you'll have black bars on the left and right sides of your video. This filter will detect and present that information in the graph reading.
 
-## 9\. Crop Height (CropH) {#croph}
+## Crop Height (CropH) {#croph}
 
 This filter enables you to view which parts, if any, of your video which contain no picture data. In the case of letterboxing, for example, you'll have black bars both above and below your picture. This filter would detect and present that in the graph reading.
 
-## 10\. Peak Signal to Noise Ratio (PSNRf) {#psnrf}
+## Peak Signal to Noise Ratio (PSNRf) {#psnrf}
 
 This filter plots the Peak Signal to Noise Ratio between the video in field 1 (odd lines) versus the video in field 2 (even lines). Lower values indicate that field 1 and field 2 are becoming more different as would happen during a playback error such as a head clog. See [http://ffmpeg.org/ffmpeg-filters.html#psnr](http://ffmpeg.org/ffmpeg-filters.html#psnr) for more information.
 
 #### Samples which demonstrate Peak Signal to Noise Ratio anomalies:
 
 ![PSNRf](media/NGA000006_psnrf.jpg)
 
-## 11\. Mean Square Error (MSEf) {#msef}
+## Packet Duration Time (pkt_duration_time) {#pkd}
+
```

Comment:
I suggest either removing the subsection with no content or adding content such as:

> The duration of the frame's presentation. Note: for constant frame rate video this will be the same value for every frame while it will vary for variable frame rate video.